revscoring/features/revision.py

```diff
@@ -45,6 +71,19 @@ def process_has_custom_comment(revision_metadata):
 has_custom_comment = Feature("revision.has_custom_comment",
                              process_has_custom_comment,
                              returns=bool, depends_on=[revision.metadata])
+"""
+Represents whether the edit has custom edit summary.
```

Comment:
It would be good to explain that "/* <something */" is not considered a custom comment because mediwiki automatically generates that.